docs/data/rpc/admin-guide.mdx

```diff
@@ -305,18 +305,18 @@ docker run -p 8001:8001 -p 8000:8000 \
 
 #### Configuration
 
-For production, we recommend running Soroban RPC with a [TOML](https://toml.io/en/) configuration file rather than CLI flags. This is similar to creating a configuration file for Stellar-Core as we did previously. For example, using [our docker image](https://hub.docker.com/r/stellar/soroban-rpc):
+For production, we recommend running Stellar RPC with a [TOML](https://toml.io/en/) configuration file rather than CLI flags. This is similar to creating a configuration file for Stellar-Core as we did previously. For example, using [our docker image](https://hub.docker.com/r/stellar/soroban-rpc):
 
 ```bash
 docker run -p 8001:8001 -p 8000:8000 \
 -v <CONFIG_FOLDER>:/config stellar/soroban-rpc \
 --config-path <RPC_CONFIG_FILENAME>
 ```
 
-You can use Soroban RPC itself to generate a starter configuration file:
+You can use Stellar RPC itself to generate a starter configuration file:
 
 ```bash
-docker run stellar/soroban-rpc gen-config-file > soroban-rpc-config.toml
+docker run stellar/soroban-rpc gen-config-file > stellar-rpc-config.toml
```

Comment:
I would leave this one, because the source code in the repo isn't actually changed yet right?